packages/@aws-cdk/aws-apigatewayv2/README.md

```diff
@@ -164,17 +164,39 @@ Note that, `HttpApi` will always creates a `$default` stage, unless the `createD
 ### Custom Domain
 
 Custom domain names are simpler and more intuitive URLs that you can provide to your API users. Custom domain name are associated to API stages.
+Learn more at [Setting up custom domain name for an HTTP API](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-custom-domain-names.html)
+and [Setting up custom domain name for a Websocket API](https://docs.aws.amazon.com/apigateway/latest/developerguide/websocket-api-custom-domain-names.html)
 
 The code snippet below creates a custom domain and configures a default domain mapping for your API that maps the
 custom domain to the `$default` stage of the API.
 
+A domain name has two important properties - DomainNameConfigurations and MutualTLS.
+
+DomainNameConfigurations are the properties associated with a DomainName - certificate, domain endpoint, security policy. Each configuration must
```

Comment:
Do you mind running this paragraph by our doc writers? Seems a bit clunky, but I don't have immediate suggestions to fix that.

Reply:
Sure, updating to the content that Will finalized.